frontend/package.json

```diff
@@ -10,8 +10,10 @@
     "emotion": "^9.2.4",
     "immer": "^1.3.1",
     "react": "^16.4.1",
+    "react-circular-progressbar": "^1.0.0",
     "react-dom": "^16.4.1",
     "react-emotion": "^9.2.4",
+    "react-markdown": "^3.6.0",
```

Comment:
For markdown rendering there is already the `react-remarkable` package in the dependencies. Can you use that please?

Reply:
Hiya jkimbo, I have found react-markdown a bit more config for instance we need to have a specific component for all links. We could remove react-remarkable instead?

e.g
<Markdown
          source={this.props.email.body}
          renderers={{
            link: props => RouterLink({ ...this.props, ...props }),
            paragraph: Paragraph,
            heading: Heading,
          }}
/>

Reply:
This was not installed in this branch so it shouldn't block this, but the master has this requirement missing from package.json. It could possibly added to this branch which removed remarkable https://github.com/cybsafe/phishtray/pull/69

Reply:
Fair enough. `react-markdown` looks good!